src/vellum/client/types/prompt_request_json_input.py

```diff
@@ -1,10 +1,9 @@
 # This file was auto-generated by Fern from our API Definition.
 
-import typing
-
+from ..core.pydantic_utilities import UniversalBaseModel
 import pydantic
-
-from ..core.pydantic_utilities import IS_PYDANTIC_V2, UniversalBaseModel
+import typing
+from ..core.pydantic_utilities import IS_PYDANTIC_V2
```

Comment:
I fixed the pre commit in a different PR such that this shouldn't be an issue going forward